Assets/MRTK/Core/Providers/InputSimulation/SimulatedHandDataProvider.cs

```diff
@@ -27,10 +27,9 @@ internal class SimulatedHandState
         // Show a tracked hand device
         public bool IsTracked = false;
         // Activate the pinch gesture
-        public bool IsPinching
-        {
-            get { return gesture == ArticulatedHandPose.GestureId.Pinch; }
-        }
+        // Pinch is a special gesture that triggers the Select and TriggerPress input actions
+        // The pinch action doesn't occur until the gesture is completed.
+        public bool IsPinching => gesture == ArticulatedHandPose.GestureId.Pinch && gestureBlending == 1.0f;
```

Comment:
Looks like this was originally changed in https://github.com/microsoft/MixedRealityToolkit-Unity/pull/7631, which allowed for the gaze-based input simulation instead of needing hands up. Mostly commenting to make sure that feature didn't regress, though I think our tests should catch if it did!

Reply:
Yeah testing it locally it works! That original change might not have been completely necessary 😆